src/wp-includes/rest-api/endpoints/class-wp-rest-font-faces-controller.php

```diff
@@ -874,14 +874,8 @@ protected function handle_font_file_upload( $file ) {
 
 		$overrides = array(
 			'upload_error_handler' => array( $this, 'handle_font_file_upload_error' ),
-			// Arbitrary string to avoid the is_uploaded_file() check applied
-			// when using 'wp_handle_upload'.
-			'action'               => 'wp_handle_font_upload',
```

Comment:
LGTM, this would cause...
`is_uploaded_file()` to be checked instead of `is_readable()`

https://github.com/WordPress/wordpress-develop/blob/45ec568f0ce905b9aabfb469b613e39d042ad3b7/src/wp-admin/includes/file.php#L928-L932

A check check for a non-zero file,
https://github.com/WordPress/wordpress-develop/blob/45ec568f0ce905b9aabfb469b613e39d042ad3b7/src/wp-admin/includes/file.php#L934-L936

and to use `move_uploaded_file()` instead of copy.
https://github.com/WordPress/wordpress-develop/blob/45ec568f0ce905b9aabfb469b613e39d042ad3b7/src/wp-admin/includes/file.php#L1013-L1020

`$action` being different doesn't affect any other filter values or be passed around, except, the `{$action}_overrides` and `{$action}_prefilter` both of which are only documented as having the following variants for `action`: `wp_handle_sideload` and `wp_handle_upload`. 
Note: Font loading from a URL should've been setting this as `wp_handle_sideload` for max compat and security checking.